queries/nasm/highlights.scm

```diff
@@ -0,0 +1,141 @@
+;; adapted from https://github.com/naclsn/tree-sitter-nasm/blob/main/queries/highlights.scm
+
+((word) @constant
+  (#match? @constant "^[A-Z_][?A-Z_0-9]+$"))
```

Comment:
ouch. is there now way to make these more explicit (if necessary, as multiple patterns)?

Reply:
More explicit?

Reply:
Using simple Lua patterns, rather than a big (Vim) regex.

Reply:
I can change it to `lua-match` (I forgot to do that) but I don't know if I can simplify it.

Reply:
Ah, ok, that would be a good compromise (I assumed that the `match` was deliberate since it couldn't be done with `lua-match`).